playbooks/allocate_pubcloud.yml

```diff
@@ -58,7 +64,7 @@
           {% for instance in rax.success %}
           {{instance.name}} ansible_host={{instance.accessIPv4}} ansible_user=root
           {% endfor %}
-        dest: '{{lookup("env", "WORKSPACE")}}/rpc-gating/playbooks/inventory/hosts'
+        dest: '{{inventory_path}}/hosts'
```

Comment:
I'm not clear on why this is necessary. Is the node this executes against not the same as the node doing the executing?